atf_plotter/scripts/plot.py

```diff
@@ -142,10 +142,8 @@ def plot_benchmark(self, style, sharey, hide_groundtruth, hide_min_max, filter_t
                         yerr = [[0], [0]]
 
                     # set marker transparency (filled or transparent)
-                    if metric_result.started\
-                        and metric_result.finished\
-                        and (metric_result.groundtruth.result or not metric_result.groundtruth.available)\
-                        and metric_result.data.stamp != rospy.Time(0):
+                    if metric_result.status == TestblockStatus.SUCCEEDED\
+                        and (metric_result.groundtruth.result or not metric_result.groundtruth.available):
```

Comment:
no boolean operation with `metric_result.groundtruth.result` - should be `(metric_result.groundtruth.result == Groundtruth.SUCCEEDED)`, right?

Reply:
fixed in https://github.com/floweisshardt/atf/pull/150/commits/24ba26a2b83488bb61a08b9d2a9447f3da58d4ff